posthog/session_recordings/queries_to_replace/sub_queries/cohort_subquery.py

```diff
@@ -10,12 +10,18 @@
 class CohortPropertyGroupsSubQuery(SessionRecordingsListingBaseQuery):
     raw_cohort_to_distinct_id = """
     SELECT
-    distinct_id
-FROM raw_person_distinct_ids
-WHERE distinct_id in (SELECT distinct_id FROM raw_person_distinct_ids WHERE 1=1 AND {cohort_predicate})
-GROUP BY distinct_id
-HAVING argMax(is_deleted, version) = 0 AND {cohort_predicate}
-    """
+        distinct_id
+    FROM
+        raw_person_distinct_ids
+    WHERE distinct_id in (
+        SELECT distinct_id
+        FROM raw_person_distinct_ids
+        WHERE 1=1 AND {cohort_predicate}
+        )
+    GROUP BY
+        distinct_id
+    HAVING argMax(is_deleted, version) = 0
```

Comment:
copied in from the CH upgrade PR - remove `AND {cohort_predicate}`
test still pass so 🙈